lib/eco/incparser/astree.py

```diff
@@ -623,10 +624,9 @@ def has_changes(self, version = None):
             return self.get_attr("changed", version) or self.get_attr("nested_changes", version)
         return self.changed or self.nested_changes
 
-    def exists(self, version = None):
-        if version:
-            return not self.get_attr("deleted", version)
-        return not self.deleted
+    def does_exist(self, version = None):
```

Comment:
Is it deliberate that the `version` parameter isn't used?

Reply:
No, forgot to remove this when I changed the method.